shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -99,7 +99,7 @@
 }
 
 void FlutterPlatformViewsController::SetFlutterView(UIView* flutter_view) {
-  flutter_view_.reset([flutter_view retain]);
+  [flutter_view_.exchange([flutter_view retain]) release];
```

Comment:
nit: what about a comment that this is retaining the new `flutter_view` and releasing the previous reference?

Reply:
That is what the one line of code says =)